devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -165,16 +168,17 @@ def parse(String description) {
 	}
 }
 
-private void createChildDevices() {    
-	if (!childDevices) {
-		def x = getChildCount()
-		for (i in 2..x) {
-			addChildDevice("Child Switch Health", "${device.deviceNetworkId}:0${i}", device.hubId,
-				[completedSetup: true, label: "${device.displayName[0..-2]}${i}", isComponent: false])
-		}
-	}
+private void createChildDevices() {   
+    if (!childDevices) {
+        def x = getChildCount()
+        for (i in 2..x) {
+            addChildDevice("Child Switch Health", "${device.deviceNetworkId}:0${i}", device.hubId,
+                           [completedSetup: true, label: "${device.displayName[0..-2]}${i}", isComponent: false])
+        }
+    }
```

Comment:
Switch back to tabs here, instead of spaces.

Reply:
@PKacprowiczS  I take the last version, it will be good